index.html

```diff
@@ -16,6 +16,18 @@ <h1>Rev My $</h1>
     </div>
     
     <h3>Hello, yourName</h3>
+
+     <!--monthly income input  -->
+     <div class="goalsAndExpenses">
+        <label for="MonthlyIncome">Monthly Income</label>
+        <input class="form-control " name="income" id="monthlyIncome" value="" autofocus placeholder="Monthly Income">
```

Comment:
please add 'required' attribute in this input's opening tag

Reply:
Working on it now. Great idea!